js/logic.js

```diff
@@ -0,0 +1,176 @@
+    technologies: ['html', 'css', 'javascript'],
+  },
+  {
+    name: 'Facebook 360',
+    workName: 'FACEBOOK',
+    position: ['Full Stack Dev', '2015'],
+    imageUrl: './img/Snapshoot Portfolio3.svg',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
+    technologies: ['html', 'css', 'javascript'],
+  },
+  {
+    name: 'Uber Navigation',
+    workName: 'Uber',
+    position: ['Lead Dev', '2015'],
+    imageUrl: './img/Snapshoot Portfolio4.svg',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and',
+    technologies: ['html', 'css', 'javascript'],
+  },
+];
+
+const populateUI = (popupOption) => {
+  popupWrapper.classList.remove('hide');
+  blur.style.filter = 'blur(3px)';
+  popupName.textContent = popupOption.name;
+  popupWorkName.textContent = popupOption.workName;
+  popupPosition.innerHTML = '';
+  popupOption.position.forEach((item) => {
+    popupPosition.innerHTML += `<li>${item}</li>`;
+  });
+  popupImage.setAttribute('src', popupOption.imageUrl);
+  popupDescription.textContent = popupOption.description;
+  popupTech.innerHTML = '';
+  popupOption.technologies.forEach((item) => {
+    popupTech.innerHTML += `<li>${item}</li>`;
+  });
+};
+
+projectOne.addEventListener('click', () => {
+  populateUI(projectDatas[0]);
+});
+
+projectTwo.addEventListener('click', () => {
+  populateUI(projectDatas[1]);
+});
+
+projectThree.addEventListener('click', () => {
+  populateUI(projectDatas[2]);
+});
+
+projectFour.addEventListener('click', () => {
+  populateUI(projectDatas[3]);
+});
+
+closePopup.addEventListener('click', () => {
+  popupWrapper.classList.add('hide');
+  blur.style.filter = 'blur(0)';
+});
```

Comment:
- [OPTIONAL] Kindly check below code snippet to know how to follow DRY principle while writing your JavaScript projects
```
   const projects = [
  {
    index: 0,
    name: 'Multi-Post Stories0',
    title: 'Gain+Glory',
    description:
      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
    image: './images/popup-image.png',
    technologies: ['ruby on rails', 'css', 'JavScript', 'html'],
    live: 'https://paulinagonzalezc.github.io/portfolio/',
    source: 'https://github.com/paulinagonzalezc/portfolio',
  },
  {
    index: 1,
    name: 'Multi-Post Stories1',
    title: 'Gain+Glory',
    description:
      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
    image: './images/popup-image.png',
    technologies: ['ruby on rails', 'css', 'JavScript', 'html'],
    live: 'https://paulinagonzalezc.github.io/portfolio/',
    source: 'https://github.com/paulinagonzalezc/portfolio',
  },
  {
    index: 2,
    name: 'Multi-Post Stories',
    title: 'Gain+Glory',
    description:
      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
    image: './images/popup-image.png',
    technologies: ['ruby on rails', 'css', 'JavScript', 'html'],
    live: 'https://paulinagonzalezc.github.io/portfolio/',
    source: 'https://github.com/paulinagonzalezc/portfolio',
  },
  {
    index: 3,
    name: 'Multi-Post Stories',
    title: 'Gain+Glory',
    description:
      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
    image: './images/popup-image.png',
    technologies: ['ruby on rails', 'css', 'JavScript', 'html'],
    live: 'https://paulinagonzalezc.github.io/portfolio/',
    source: 'https://github.com/paulinagonzalezc/portfolio',
  },
  {
    index: 4,
    name: 'Multi-Post Stories',
    title: 'Gain+Glory',
    description:
      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
    image: './images/popup-image.png',
    technologies: ['ruby on rails', 'css', 'JavScript', 'html'],
    live: 'https://paulinagonzalezc.github.io/portfolio/',
    source: 'https://github.com/paulinagonzalezc/portfolio',
  },
  {
    index: 5,
    name: 'Multi-Post Stories',
    title: 'Gain+Glory',
    description:
      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
    image: './images/popup-image.png',
    technologies: ['ruby on rails', 'css', 'JavScript', 'html'],
    live: 'https://paulinagonzalezc.github.io/portfolio/',
    source: 'https://github.com/paulinagonzalezc/portfolio',
  },
];

const createCard = (project) => {
  const ul = document.createElement('ul');
  ul.className = 'languages';
  let liList = '';
  for (let i = 0; i < project.technologies.length; i += 1) {
    liList += `<li class="language">${project.technologies[i]}</li>`;
  }

  ul.innerHTML = liList;

  const tmp = document.createElement('div');
  tmp.appendChild(ul);
  const ulStirng = tmp.innerHTML;
  const card = `
          <div class="work-card">
            <div class="card-content">
              <h3 class="card-title1">${project.name}</h3>
              <h3 class="card-title2">${project.title}</h3>
              ${ulStirng}
              <div class="button-container">
                <button id="btn-project-${project.index}" data-modal-target="#modal" class="project-button">See Project</button>
              </div>
            </div>
          `;

  projectHolder.innerHTML += card;
};

const populatePage = (arr) => {
  for (let i = 0; i < arr.length; i += 1) {
    createCard(arr[i]);
  }
};

populatePage(projects);
```